config.plist/alder-lake.md

```diff
@@ -40,7 +40,6 @@ For us we'll need a couple of SSDTs to bring back functionality that Clover prov
 | **[SSDT-PLUG-ALT](https://dortania.github.io/Getting-Started-With-ACPI/)** | Adds ACPI `Processor` objects since macOS does not support `Device` objects used on newer boards, see [Getting Started with ACPI](https://dortania.github.io/Getting-Started-With-ACPI/) for more details. |
 | **[SSDT-EC-USBX](https://dortania.github.io/Getting-Started-With-ACPI/)** | Fixes both the embedded controller and USB power, see [Getting Started with ACPI](https://dortania.github.io/Getting-Started-With-ACPI/) for more details. |
 | **[SSDT-AWAC](https://dortania.github.io/Getting-Started-With-ACPI/)** | This the 300 and 600 series RTC patch, required for all B660 and Z690 boards. The alternative is SSDT-RTC0 or SSDT-AWAC, depending on the ACPI definitions in your DSDT, see [Getting started with ACPI](https://dortania.github.io/Getting-Started-With-ACPI/) for more details. |
-| **[SSDT-BRG0](https://dortania.github.io/Getting-Started-With-ACPI/)** | May not be needed for everyone. Fixes some devices being missing from your ACPI tables, see [Getting started with ACPI](https://dortania.github.io/Getting-Started-With-ACPI/) for more details. |
```

Comment:
SSDT-BRG0 is required for newer RX 6900 XT cards with XTXH chips. Without this SSDT device properties injection will not work.

Reply:
Hadn't thought of that. I removed it because usually the guide here only mentions SSDTs that are required to boot, and anything else is a post-install step. I'll add it back, thanks.